lib/trlbuf.c

```diff
@@ -637,61 +626,105 @@ void CloseTrlBuf (struct TrlBuf * buf)
         status = fcloseWithStatus(&buf->fp);
 
 }
-void trlmessage (const char *message) {
+void trlmessage(const char *fmt, ...) {
+    va_list args;
+    va_start(args, fmt);
+
+    char *fmt_ = calloc(strlen(fmt) + 2, sizeof(*fmt_));
+    if (!fmt_) {
+        perror("trlwarn: calloc failed");
+        exit(1);
```

Comment:
Do we not want to use one of them codes in `hstcalerr.h`?

Reply:
That's done